bucket/super-productivity.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "6.1.2",
+    "description": "To-do list & time tracker for programmers and other digital workers with Jira, Github, and Gitlab integration.",
+    "homepage": "https://super-productivity.com/",
+    "license": "MIT",
+    "url": "https://github.com/johannesjo/super-productivity/releases/download/v6.1.2/superProductivity-Setup-6.1.2.exe#/dl.7z",
+    "hash": "bc7523cffb1e9d64ae7ab0760c7c4432b743cfa5503c386de2a01f7da554c664",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse"
+                ]
+            }
+        }
+    },
+    "uninstaller": {
+        "file": "Uninstall superProductivity.exe"
+    },
+    "bin": "superProductivity.exe",
```

Comment:
What are avialable parameters?

Reply:
I don't think there are any input parameters.

Reply:
Then 

```suggestion
```

Reply:
Got it

Reply:
If I remove the bin prop I receive the error "The term 'superproductivity' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a
path was included, verify that the path is correct and try again."

Reply:
That is the point. There is not a single reason to start the gui applications from command line, when there are no parameters....